frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -499,13 +501,32 @@ export function useFilterDimensions(version: 1 | 2 = 1): {
         return [
           {
             id: collection.id,
-            name: collection.summaryCitation || "",
+            name: collection.summaryCitation || "No Publication",
```

Comment:
might be nice to have the string `"No Publication"` set as a variable somewhere so you can reference it throughout this file

Reply:
+1

Reply:
Agreed! That change is reflected in the "No Publication" PR. Here: https://github.com/chanzuckerberg/single-cell-data-portal/pull/5047